hazelcast/src/main/java/com/hazelcast/client/impl/protocol/map/MapMessageType.java

```diff
@@ -0,0 +1,29 @@
+package com.hazelcast.client.impl.protocol.map;
+
+import com.hazelcast.client.impl.protocol.ClientMessageType;
+
+/**
+ * Map message types
+ */
+public enum MapMessageType {
+
+    /**
+     * Map put
+     */
+    MAP_PUT(9),
```

Comment:
MessageType id is (will be) documented in ClientProtocol . So those ids shouldn't be modified. As there will be around 500 messages I'm planing to organise these by distributed object types like map.
Fragility shouldn't  be an issue as they are validated against our client protocol.